src/benchmarks/micro/README.md

```diff
@@ -1,287 +1,90 @@
-# Benchmarks
+# Micro Benchmarks
 
-This repo contains various .NET benchmarks. It uses BenchmarkDotNet as the benchmarking engine to run benchmarks for .NET, .NET Core, CoreRT and Mono. Including private runtime builds.
+This folder contains micro benchmarks that test the performance of .NET Runtime(s).
 
-## BenchmarkDotNet
+## Tooling
 
-Benchmarking is really hard (especially microbenchmarking), you can easily make a mistake during performance measurements.
-BenchmarkDotNet will protect you from the common pitfalls (even for experienced developers) because it does all the dirty work for you:
+To run the benchmarks, you need to [download dotnet cli](../../../docs/prerequisites.md) first.
 
-* it generates an isolated project per runtime
-* it builds the project in `Release`
-* it runs every benchmark in a stand-alone process (to achieve process isolation and avoid side effects)
-* it estimates the perfect invocation count per iteration (based on `IterationTime`)
-* it warms-up the code
-* it evaluates the overhead
-* it runs multiple iterations of the method until the requested level of precision is met
-* it consumes the benchmark result to avoid dead code elimination
-* it prevents from inlining of the benchmark by wrapping it with a delegate.
+We use BenchmarkDotNet as the benchmarking tool, you can read more about it in [our short summary](../../../docs/benchmarkdotnet.md) (it's recommended). The key thing that you need to remember is that **BenchmarkDotNet runs every benchmark in a dedicated process and stops the benchmarking when a good level of precision is met**.
```

Comment:
"good level" is nebulous. perhaps something like "specified level"? If there's a good doc about how BDN does this a link to it here would be great.